pyV2DL3/constant.py

```diff
@@ -1,4 +1,4 @@
-# Global setting for both VEGAS and ED
+# Global setting for both VEGAS and Eventdisplay
 
 VERSION = "0.2"
```

Comment:
Good point - wasn't even aware that the version is written in there. Should find a better solution in future. Set it now to `0.4.0-beta`, as code has changed since 0.3.

Reply:
I move it back to 0.2, as otherwise the CI fails. Opened an issue that version numbers should be treated consistently in future: #162

Reply:
We can update the test files for CI or?

Reply:
Yes, but only with the release.